main.go

```diff
@@ -39,7 +38,7 @@ func main() {
 
 	global.GithubClient, err = github.Create()
 
-	global.DiscordClient, err = discordgo.New("Bot " + global.Token)
+	global.DiscordClient, err = discord.Create()
```

Comment:
i think this works for now but long term i would like to come up with a way to not use globals and instead package level access to vars and methods.

Reply:
Agreed, I'll create a card for this